presto-spi/src/main/java/com/facebook/presto/spi/type/CharType.java

```diff
@@ -30,6 +30,7 @@
         extends AbstractVariableWidthType
 {
     public static final int MAX_LENGTH = 65_536;
+    public static final CharType CHAR = new CharType(MAX_LENGTH);
```

Comment:
do we need to introduce unparametrized Char type instance to SPI just to use it in test? Maybe just instantiate char type directly in test, e.g: use `createCharType(MAX_LENGTH)`?